tests/test_serialization.py

```diff
@@ -123,3 +124,117 @@ def test_run_eagerly():
         model__hidden_layer_sizes=(100,),
     )
     check_pickle(estimator, load_boston)
+
+
+def _weights_close(model1, model2):
+    w1 = [x.numpy() for x in model1.model_.weights]
+    w2 = [x.numpy() for x in model2.model_.weights]
+    assert len(w1) == len(w2)
+    for _1, _2 in zip(w1, w2):
+        assert np.allclose(_1, _2, rtol=0.01)
+    return True
+
+
+def _reload(model, epoch=None):
+    tmp = pickle.loads(pickle.dumps(model))
+    assert tmp is not model
+    if epoch:
+        assert tmp.current_epoch == model.current_epoch == epoch
+    return tmp
+
+
+@pytest.mark.parametrize(
+    "optim", ["adam", "sgd", keras.optimizers.Adam(), keras.optimizers.SGD(),],
+)
+def test_partial_fit_pickle(optim):
```

Comment:
Does this test pass now for all optimizers?

Reply:
Yep!

Reply:
Working now except on Windows. I'll need to get a Windows computer to test with.